revolv/base/management/commands/seed.py

```diff
@@ -99,7 +99,8 @@ class ProjectSeedSpec(SeedSpec):
         "location": "2415 Bowditch St, Berkeley, CA 94704, United States",
         "location_latitude": 37.8670289,
         "location_longitude": -122.2561597,
-        "end_date": datetime.date(2050, 10, 8),
+        "start_date": datetime.date(2014, 1, 1),
+        "end_date": datetime.date(2015, 1, 1),
```

Comment:
Nah, I think it's good to have projects that are in the middle of running, because then we can see days left, total days, days so far as different numbers so as not to get confused between them.

I actually think the end date should be relative though, so we don't have to update it. I will put that in now